resources/views/tasks/show.blade.php

```diff
@@ -1,14 +1,144 @@
 @extends('layouts.layout')
 
 @section('content')
-<div class="container" id="task">
-    <task-view
-        process-uid="{{$process->uid}}" instance-uid="{{$instance->uid}}" token-uid="{{$token->uid}}" form-uid="{{$token->definition['formRef']}}"
-        :data="{{json_encode($data)}}" token-created="{{$token->delegate_date}}" token-completed="{{$token->finish_date}}"
-        user-uid="{{$token->user->uid}}" user-name="{{$token->user->fullname}}" user-avatar="{{$token->user->avatar}}"></task-view>
+<div>
+    <nav>
```

Comment:
Does this need to be a nav bar? Can it be accomplished using the grid instead?

Reply:
converted to grid